EDA.ipynb

```diff
@@ -2,34 +2,36 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 12,
+   "execution_count": 40,
    "metadata": {
     "collapsed": true
    },
    "outputs": [],
    "source": [
+    "import random\n",
     "import pandas as pd\n",
     "import numpy as np\n",
-    "import matplotlib as plt\n",
+    "import matplotlib.pyplot as plt\n",
+    "import matplotlib.image as mpimg\n",
     "import seaborn as sns\n",
     "\n",
     "%matplotlib inline"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 13,
+   "execution_count": 41,
    "metadata": {
     "collapsed": true
    },
    "outputs": [],
    "source": [
-    "data = pd.read_csv('fashionmnist/fashion-mnist_train.csv')"
+    "data = pd.read_csv('/fashionmnist/fashion-mnist_train.csv')"
```

Comment:
remove spurious '/' at beginning of file path

Reply:
Ok, will do! Thanks!